LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
Do you have a specific interest in adding the LICENSE explicitly?
We normally do not do this for our packages...